packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -40,6 +56,22 @@ export interface AcceleratorProps {
    * @default true
    */
   readonly enabled?: boolean;
+
+  /**
+   * IP addressess associated with the accelerator.
```

Comment:
```suggestion
   * IP addresses associated with the accelerator.
```

Reply:
Also we would be good to add an additional description based on the doc.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-globalaccelerator-accelerator.html#aws-resource-globalaccelerator-accelerator-properties

```
Optionally, if you've added your own IP address pool to Global Accelerator (BYOIP), you can choose IP addresses from your own pool to use for the accelerator's static IP addresses when you create an accelerator. You can specify one or two addresses, separated by a comma. Do not include the /32 suffix.

Only one IP address from each of your IP address ranges can be used for each accelerator. If you specify only one IP address from your IP address range, Global Accelerator assigns a second static IP address for the accelerator from the AWS IP address pool.

Note that you can't update IP addresses for an existing accelerator. To change them, you must create a new accelerator with the new addresses.
```